raylib.pc.in

```diff
@@ -1,7 +1,7 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-exec_prefix=${prefix}
```

Comment:
Why is changing this line necessary?

Reply:
This was copy pasted from https://github.com/fmtlib/fmt/blob/master/support/cmake/fmt.pc.in
I've reverted the change.